lib/components/MapHeatmap.js

```diff
@@ -62,10 +65,14 @@ class MapHeatmap extends React.Component {
 
   render() {
     const AIRMapHeatmap = this.getAirComponent();
-
+    let points = this.getSanitizedPoints()
+    // Exception thrown on android if 0 points are provided.
+    if (points.length < 1) {
+      points = points.concat([{latitude: 0, longitude: 0, weight: 0}])
```

Comment:
Concatenating is going to create a whole new array instance and add a single item.  Can't you just call `.push({latitude: 0, longitude: 0, weight: 0});` ???